Moderation_Matrix.md

```diff
@@ -0,0 +1,61 @@
+# Moderation Matrix
+This matrix correlates with the [Community Rules](https://www.theodinproject.com/community_rules) that are required to be followed in order to participate in the community. Some rules have different levels of severity, so you may see references to them under different numbers of zaps. 
+
+This matrix is not all encompassing and zaps are given at the discretion of the moderator team.
+
+## 0 Zaps (Heads Up)
+- Dog piling/short circuiting a teaching moment
+- Asking about or encouraging the use of Windows
+- "Chat bombing", aka posting many short (ususally 4-5 or fewer words) messages in rapid succession, repeatedly sending parts of thoughts (incomplete, non-standalone sentences or fragments) in new messages. (Use shift + enter and send ONE message where possible or edit your message) 
+- Discussing mental health issues in depth
+
+## 1 Zap (WARNING) 
+Minor violations such as:
+- A second infraction of any 0 Zap issues above
+- "Mini-modding"
+  - Please leave moderation action to the moderators, use ModMail to report rule breaking or if you see something you’re unsure about in the community
+- Asking about topics or projects unrelated to The Odin Project 
+- Mild unprofessionalism:
+  - Drug related conversation 
+  - Treating others on the server disrespectfully
+  - Mental health issues discussion 
+  - Relationship advice
+
+## 2 Zaps (WARNING)
+Medium severity violations such as: 
+- Suggesting piracy 
+- Unsolicited pings/DM/friend request 
+- Political or religious topics 
+- Mild toxicity:
+  - Directed insults
+  - Provoking/baiting other users
+  - Passive aggression
+  - Naming & shaming
+- Self promo without permission (ask with Modmail)
+  - Do not share resources you have created solely for personal or monetary gain
+  - Examples: "Fill out my survey" (does not help the goals of Odin), "Join my boot camp/program/server" (Could mislead learners), "Why PHP is superior" (unrelated blog post)
+
+## 5 Zaps (SEVERE WARNING)
```

Comment:
Did we decide against doing temp bans and/or timeouts at certain zap levels pre-permaban?

Reply:
Nope, implementation itself has not been determined.

Reply:
I could be in favor of this level of zaps resulting in a temp ban (length TBD). At this point, a user would have accumulated enough zaps and thus had enough warnings regarding behavior, that a temp ban would give them time to reflect without causing any further issues on the server.

I will say I can also see the drawback to this. If someone has had zero issues before and out of nowhere links to a pirated source, should this first ever offense of a user result in a temp ban?

Reply:
Yeah that's my thing, it would be difficult to be consistent here.

Reply:
I don't think this is something that we need to hammer out before we can move forward by any means. I also wouldn't be against temp bans just being a matter of moderator discretion.